### PR TITLE
Attendance Tracker Fixes

### PR DESCRIPTION
**Pull request by yottalogical**
# Checklist

**This pull request is a hotfix**

  * [x] This patch is under 50 lines total.
  * [x] This patch contains only small content additions or critical
        functionality, layout, or style fixes.
  * [x] This patch follows the FAMNM Website Code Style Guide.
  * [x] This patch makes no changes to existing functionality, layout, or style
        rules, beyond the issue it is intended to fix.
  * [x] If the patch fixes a functionality, layout, or style issue, the patch
        has been tested on both desktop and mobile platforms.

# Description

 * Doesn't show a warning about uniqname format if the uniqname field is empty.
 * Modal hides correctly after submitting.
 * Gives some info about green uniqnames on the confirmation page.
 * Does a final check that a meeting type has been selected right before submission.

### CODE REVIEW

File: attendance/index.html [hunk 1]
```diff
@@ -254,7 +262,7 @@ <h2>Meeting Attendees</h2>
                     $("#meetingSuccessAlert").show();
                     $("#meetingFailAlert").hide();
 
-                    confirmModal.hide();
+                    $("#confirmMeeting").modal("hide");
```
Comment (yottalogical) [inline attendance/index.html:265]: Here!

File: attendance/index.html [hunk 2]
```diff
@@ -267,7 +275,7 @@ <h2>Meeting Attendees</h2>
                 $("#meetingSuccessAlert").hide();
                 $("#meetingFailAlert").show();
 
-                confirmModal.hide();
+                $("#confirmMeeting").modal("hide");
```
Comment (yottalogical) [inline attendance/index.html:278]: Here!

**Review by yottalogical — COMMENTED** [review]
I've changed this back to the way it was before [the Bootstrap upgrade](https://github.com/FAMNM/famnm-website/pull/171/files#diff-41633ae0233e7d13044211c367b4057362495d81868fd79f1014223c61039e17L237) since the new way didn't seem to work, but the old way is working just fine.

However, there might be some other thing that I'm not considering, so I wanted to call your attention to this change just in case.

**Comment by yottalogical** [comment]
Also, I had to update the `Gemfile.lock` to get Jekyll working on my machine. It may be best to doublecheck that everything still works for you too.

**Comment by broad-well** [comment]
The new Gemfile.lock didn't break my local setup, and I verified all the fixed behavior while recording my attendance at the website meeting today. :+1:

**Review by broad-well — APPROVED** [review]

**Pull request merged**